packages/compiler-sfc/__tests__/compileScript.spec.ts

```diff
@@ -1705,5 +1705,20 @@ describe('SFC analyze <script> bindings', () => {
       expect(content).toMatch(`name: 'Baz'`)
       assertCode(content)
     })
+
+    test('defineProps starts with parenthesis', () => {
```

Comment:
I think the position of the test is not correct.

Maybe it should be moved to line 98